modules/draco/src/lib/draco-parser.js

```diff
@@ -217,19 +219,14 @@ export default class DracoParser {
   _getMeshFaceIndices(decoder, dracoGeometry) {
     // Example on how to retrieve mesh and attributes.
     const numFaces = dracoGeometry.num_faces();
-
     const numIndices = numFaces * 3;
-    const indices = new Uint32Array(numIndices);
-    const dracoArray = new this.draco.DracoInt32Array();
-    for (let i = 0; i < numFaces; ++i) {
-      decoder.GetFaceFromMesh(dracoGeometry, i, dracoArray);
-      const index = i * 3;
-      indices[index] = dracoArray.GetValue(0);
-      indices[index + 1] = dracoArray.GetValue(1);
-      indices[index + 2] = dracoArray.GetValue(2);
-    }
+    const byteLength = numIndices * INDEX_ITEM_SIZE;
+
+    const ptr = this.draco._malloc(byteLength);
+    decoder.GetTrianglesUInt32Array(dracoGeometry, byteLength, ptr);
+    const indices = new Uint32Array(this.draco.HEAPF32.buffer, ptr, numIndices).slice();
```

Comment:
Why do we `slice`? That creates a copy?

Reply:
Wrapped `this.draco.HEAPF32.buffer` have to be disposed. A copy is necessary